go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0 // indirect
 	github.com/pborman/uuid v1.2.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
FYI this package is sort of dead now. The errors.WithMessage that you're using should probably be fmt.Errorf with a %w "verb"

Reply:
Thank ye, fixed.